### PR TITLE
Corrected nats documentation

### PR DESCRIPTION
**Pull request by girishg4t**
Made changes to the nats documentation so that the keda connector should work by following the steps

### CODE REVIEW

**Comment by RealHarshThakur** [comment]
@girishg4t  
Some nitpicks(Mostly related to assuming people will not clone the repo): 
* Can you please provide links to the YAML files ?
* Can you provide the Dockerfile , Go files as code snippets in the docs, so people can copy , paste ?
* I think `deployment.yaml` is missing in the repo too, can you please check?

**Comment by girishg4t** [comment]
> Some minor changes:
> 
>     * Can you add the Dockerfile in a code snippet too?
> 
>     * Can you provide the link to deployment.yaml in kubectl command?

1) added dockerfile
2) Also added link to deplyment.yaml and code snippit

File: docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md [hunk 1]
```diff
@@ -31,26 +23,75 @@ NATS streaming keda connector uses NATS monitoring to scale the deployment, to e
...
 
-$ nats-server -m 8222
 ```
+{{% /notice %}}
 
-To verify if the nats streaming server is up and running check below url
-```bash
-$ curl http://localhost:8222/streaming/channelsz?channel=request
+```sh
+$ kubectl apply -f nats-dep.yaml
```
Comment (RealHarshThakur) [inline docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md:30]:
```suggestion
$ kubectl apply -f https://github.com/fission/keda-connectors/blob/master/nats-streaming-http-connector/test/nats-streaming-server/nats-dep.yaml
```

**Comment by RealHarshThakur** [comment]
Thanks @girishg4t

**Review by RealHarshThakur — APPROVED** [review]

**Comment by sanketsudake** [comment]
Closing this PR as changes got merged via https://github.com/fission/docs.fission.io/pull/294

**Pull request closed without merging**